framework/src/main/java/org/checkerframework/framework/util/FlowExpressionParseUtil.java

```diff
@@ -158,21 +153,102 @@ private static Pattern anchored(String regex) {
     }
 
     private static boolean isMemberSelect(String s, FlowExpressionContext context) {
-        return parseMemberSelect(s) != null;
+        return parseMemberSelect(s, context) != null;
+    }
+
+    /**
+     * Matches field accesses whose receiver is a method call expression. First of returned pair is
+     * a method call expression and the second is a field or method call expression.
+     *
+     * @param sWithFormalParamNames expression string with replaced occurences of one-based
+     *     parameter index of with formal parameter names
+     * @param s expression string
+     * @param argumentList list of formal parameter names corresponding to index of formal
+     *     parameters in the method call that is in s
+     * @return pair of method call expression and field, or two method call expressions
+     */
+    private static Pair<String, String> parseMethodCall(
```

Comment:
I still have the concern, which I mentioned in my previous review, that the signature of this method is different than that of other `parseMethodCall` methods,  This seems to be parsing not a method call, but an expression whose receiver is a method call.  Please clarify the documentation.